07_system_overview.R

```diff
@@ -983,7 +996,7 @@ universe <- reactive({
             
             # nbn shelter
             (ProjectType == es_nbn_project_type &
-               (in_date_range == TRUE | NbN15DaysAfter == TRUE)) |
+               (in_date_range == TRUE | NbN15DaysAfterReportEnd == TRUE)) |
```

Comment:
for consistency, I'd use `NbN15DaysAfterReportEnd == 1` instead of `NbN15DaysAfterReportEnd == TRUE`, since prior to this we are setting NbN15DaysAfterReportEnd to 1s and 0s, not TRUE and FALSE. But I know they resolve to the same thing, so nbd.

Reply:
In this branch, I'm showing those are resolving to TRUE and FALSE not 1 and 0 so we'll leave it for now- thanks!